_schemaV2.graphql

```diff
@@ -10225,13 +10226,23 @@ type EditionSet implements Sellable {
   internalID: ID!
   isAcquireable: Boolean
   isForSale: Boolean
+
+  # Is this artwork part of an auction?
```

Comment:
Super #nitpick but might be worth not to use artwork here as we are looking at edition set and the info is read from artwork but does not exist on EditionSet itself. So maybe 'is this edition set belongs to the artwork in an auction' is more accurate description? Or just `Is this object part of an auction?`